torch/testing/_internal/distributed/rpc/rpc_test.py

```diff
@@ -1384,6 +1436,30 @@ def test_rref_str(self):
             ),
         )
 
+    @dist_init
+    def test_rref_get_future(self):
+        # Tests that we can obtain the future corresponding to the creation of
+        # the RRef on remote end
+        if self.rank == 0:
+            # Builtin
+            rref = rpc.remote(worker_name(1), torch.add, args=(1, 1))
+            rref.to_here()
+            fut = rref._get_future()
+            self.assertIsInstance(fut, torch.distributed.rpc.Future)
```

Comment:
We can test this future a little more by calling `wait_until_pending_users_flushed()` and then ensuring that the future is completed, although, there's no pythonAPI to check if a future is already completed. Not sure if it's worth adding for the test.